packages/repository/src/repositories/legacy-juggler-bridge.ts

```diff
@@ -148,6 +148,21 @@ export class DefaultCrudRepository<
       return model as typeof juggler.PersistedModel;
     }
 
+    return this.definePersistedModel(entityClass);
```

Comment:
Why don’t we emit an event when definePersistedModel is done so that listeners can be registered to set up operation hooks? Overriding a method is always more advanced for a lot of developers.

Reply:
For the record, we briefly discussed if it's feasible to use events to set up operation hooks and decided to defer that.